DQM/L1TMonitorClient/python/L1TStage2uGTClient_cff.py

```diff
@@ -4,21 +4,59 @@
 # directory path shortening
 ugtDqmDir = 'L1T/L1TStage2uGT'
 # input histograms
-errHistNumStr = 'errorSummaryNum'
-errHistDenStr = 'errorSummaryDen'
+errHistNumStr = 'first_bunch_in_train'
```

Comment:
Change this back. It is needed for l1tStage2uGMTOutVsuGTInRatioClient but not for your dqmRatioTimingPlots.